CONTRIBUTING.md

```diff
@@ -283,3 +283,93 @@ The file structures in this repository should look like this :
 
 ## Join The Project Team
 <!-- TODO -->
+
+## Adding an API endpoint
+
+This example will use `users` as a new TAG for OpenAPI, meaning we will have dedicated endpoints to get some mocked users or create new ones.
+
+### How to add an endpoint
+
+Endpoints should be created in a `<TOPIC>-routes.ts` file under `./modules/<TOPIC>/api` folder.
+We assume `<TOPIC>` as a new TAG on OpenAPI, meaning a different category with dedicated endpoints like `address`, `countries`, `currency`, and son on... We will use `users` topic in our examples.
+
+Example on how to create and endpoint for a new topic:
+
+1 - Create a new folder with the topic name, under `./modules` folder, like `./modules/users`
+2 - Create a new file with mocked data, under `./modules/users/data`, like `./modules/users/data/users.ts`. Example:
+
+```javascript
+const usersList = [
+    {
+    "email": "male@example.com",
+    "gender": "male",
+    "username": "user0000",
+    "first_name": "John",
+    "last_name": "Doe",
+    "title": "mr",
+    },
+    {
+    "email": "female@example.com",
+    "gender": "male",
+    "username": "user0001",
+    "first_name": "Hydra",
+    "last_name": "Smith",
+    "title": "mrs",
+    },
+    ...
+]
+```
+
+3 - Create an `api` folder to store the `routes` file, like `./modules/users/api/users-routes.ts`
+4 - Create different endpoints for users. Example:
+
+```javascript
+import usersList from '../data/users';
+
+module.exports = function (app: core.Express) {
+    app.get('/users', (req: Request, res: Response) => {
+        res.json({
+            users: usersList,
+        });
+    });
+}
+```
+
+Note: This endpoint will fetch all mocked users stored in `./modules/users/data/users.ts`
+
+### How our folders are structured
+
+    Mocked-API
```

Comment:
We have already just done a new pass on what the folder structure will be and the naming conventions and has recently been added to this file on dev. 👍 If you could point to that would be ace.

Reply:
Can you also if you are able move the commit message standards from the README.md file into this file.

Reply:
Ok, i will :)

Reply:
Thank you. Great work

Reply:
Thanks @ageddesi  ;)
One more question.
Should i reference "How to add tests" to the "Testing" section of `README.md` file as well?

A screenshot of the `README.md` file

![image](https://user-images.githubusercontent.com/66181027/196760421-aaf2a862-c09d-460d-a244-01f38f7544e4.png)

Reply:
Hi @bpvcode best to move testing into the contributing guide as well.